src/_scss/pages/agency/_overview.scss

```diff
@@ -105,6 +105,25 @@
                 height: rem(20);
                 width: 100%;
             }
+
+            h4 {
+                position: relative;
+
+                a {
+                    height: rem(16);
+                    width: rem(16);
+                    margin: 0 rem(6) 0 rem(2);
+                    display: inline-block;
+
+                    svg {
+                        position: absolute;
+                        top: rem(6);
+                        height: rem(16);
+                        width: rem(16);
+                        fill: $color-gray-lighter;
```

Comment:
I'd make this `$color-gray-light` for more contrast

Reply:
Done